src/emmetHelper.ts

```diff
@@ -525,10 +525,10 @@ export function getExpandOptions(syntax: string, emmetConfig?: object, filter?:
 	};
 }
 
-export function expandAbbreviationWithDash(abbreviation: string, options: any) {
+export function expandCSSAbbreviationWithVendorPrefixes(abbreviation: string, options: any) {
```

Comment:
- no need to export this function
- rename (again, sorry!) to just `expandCSSAbbreviation`